evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -436,6 +437,33 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     return (oracles[_oracle].latestAnswer, oracles[_oracle].lastReportedRound);
   }
 
+  /**
+   * @notice allows non-oracles to request a new round
+   */
+  function startNewRound()
+    external
+    onlyWhitelistedRequesters()
+  {
+    uint32 current = reportingRoundId;
+
+    bool eligible = rounds[current].updatedAt > 0 || timedOut(current);
+    require(eligible, 'Cannot start a round mid-round');
```

Comment:
Would it be cheaper to do something like:
```
require(rounds[current].updatedAt > 0 || timedOut(current), 'Cannot start a round mid-round');
```
Since we don't do anything else with `eligible`?

Reply:
Probably a bit cheaper, but I thought this would be a bit more readable.